cpp/ql/src/semmle/code/cpp/rangeanalysis/Bound.qll

```diff
@@ -76,4 +76,6 @@ class ValueNumberBound extends Bound, TBoundValueNumber {
   override string toString() { result = vn.getExampleInstruction().toString() }
 
   override Location getLocation() { result = vn.getLocation() }
+
+  ValueNumber getValueNumber() { result = vn }
```

Comment:
The Pull-Request-Check/QLDoc/CPP test is failing because this new public-visible predicate has no QLDoc. Suggestion:

```
/** Gets the value number that equals this bound. */
```

Actually, the `getInstruction` predicate above has outdated QLDoc. If you don't mind, can you change it to

```
/** Gets an `Instruction` that equals this bound. */
```